spec/requests/session_controller_spec.rb

```diff
@@ -2419,7 +2572,7 @@ def post_login
     end
 
     it 'marks the challenge as successful if the 2fa succeeds' do
-      post "/session/2fa/test-action", params: { redirect_path: "/ggg" }
+      post "/session/2fa/test-action", params: { redirect_url: "/ggg" }, xhr: true
```

Comment:
Hmm this is tricky... I'd like to support both relative paths and full URLs here. So I guess I can't use `path` or `url` in the property name. Maybe `redirect_location` is general enough to fit both paths and URLs?

Reply:
@danielwaterworth the `redirect_path` field tells the 2fa page where the user should be redirected after they confirm their 2FA. Up until this point, we've only needed the 2fa page to redirect users to another page within the site, so the name `redirect_path` made perfect sense. However, with this change, we need the page to redirect users to an external site (the site that's using Discourse Connect) so the name `redirect_path` no longer fits.

The best name that I can think of for this field is `redirect_location`. Any better names?

Reply:
Hm, paths are relative URLs, so `redirect_url` is fine imo. Just make sure there are specs that use both absolute and relative URLs so that we have examples of both usages. 😃

Reply:
Alright then! There are lots of tests that cover both cases so I think we're good here! Gonna mark the other comments as resolved since they're are all related to this.